app/helper/RTMediaLikeNotification.php

```diff
@@ -81,7 +81,10 @@ function format_like_notifications( $params ) {
 
 			$rtmedia_id = rtmedia_id( $post_id );
 			$liked_list = $this->fetch_media_like_stats( $rtmedia_id );
-			$liked_by   = bp_core_get_username( $liked_list[0]->user_id );
+                        $liked_by   = '';
```

Comment:
check where `$liked_by` is used make sure given default value doesn't cause other implications, also align variable.